VisualizationBase/src/ViewItemManager.cpp

```diff
@@ -121,6 +124,57 @@ void ViewItemManager::removeAllViewItems()
 	currentViewItem_ = nullptr;
 }
 
+ViewItem* ViewItemManager::createIfNotExists(const QString name, QPoint position)
+{
+	//Try seeing if it is opened already
+	if (auto view = viewItem(name))
+		return view;
```

Comment:
Here and everywhere else, prefer putting very short `if`/`for`/`while` bodies on the same line as the statement header.